src/Angor/Client/Pages/Signatures.razor

```diff
@@ -785,11 +842,27 @@
             notificationComponent.ShowErrorMessage("Failed to retrieve NSEC: " + e.Message);
         }
     }
+
+    private string GetShortenedNpub(string npub)
```

Comment:
this code is being duplicated perhaps a helper method in `NostrConversionHelper`?